src/amo/components/AddonMoreInfo/index.js

```diff
@@ -170,6 +174,29 @@ export class AddonMoreInfoBase extends React.Component<InternalProps> {
       );
     }
 
+    let categories = null;
+    if (
+      [ADDON_TYPE_EXTENSION, ADDON_TYPE_STATIC_THEME].includes(addon.type) &&
```

Comment:
Use `visibleAddonTypeIsValid`?

Reply:
I considered it but `visibleAddonTypeIsValid` is a very old helper function that we should kill with fire eventually. It is very complex for no good reason and it was needed when React Router didn't have support for regular expressions in URL patterns (that and the whole concept of "visible type").